Sources/Client/NetClient.cpp

```diff
@@ -714,6 +716,25 @@ namespace spades {
 			}
 		}
 
+		void NetClient::HandleExtensionPacket(spades::client::NetPacketReader& reader) {
+			std::unordered_map<uint8_t, uint8_t> implementedExtensions ( {{192, 1}, {120, 1}} );
```

Comment:
Add `static const` because it's constant.

Reply:
Sorry, it seems I messed up the patch once again. This wasn't supposed to be here.